recipes/pytest-alembic/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "pytest-alembic" %}
+{% set version = "0.3.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pytest-alembic-{{ version }}.tar.gz
+  sha256: 0670a71d5a8feb675d004ce9d4e23b583fe66a2cbdf84b88b936712d24fb43f4
+
+build:
+  number: 0
+  skip: true   # [py>=40 or py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
```

Put version spec back:
```suggestion
    - python >=3.6
```